packages/interactivity/src/utils.ts

```diff
@@ -51,20 +66,30 @@ function createFlusher( compute, notify ) {
 	return { flush, dispose };
```

Comment:
This is a nice spot for a const assertion.
```suggestion
	return { flush, dispose } as const;
```